### ISSUE TITLE
New mini inbox view's New Message icon conflicts with the app jewels

### ISSUE THREAD

**Issue by chenglou** [rsms/fb-mac-messenger#177]
This is a new feature we've implemented:

<img width="134" alt="screenshot 2015-08-13 22 59 02" src="https://cloud.githubusercontent.com/assets/1909539/9268068/eb57eafe-420e-11e5-821c-fa98a4263ab5.png">

**Comment by rsms** [comment]
Known issue. main.js needs patching.

**Comment by jasonwoodland** [comment]
if you inject custom css this seems to work:

``` css
@media (max-width: 640px) {
    ._30yy._4kzv {
        display: none;
    }
}
```

**Comment by rsms** [comment]
Fixed in release 0.1.6
